src/Cart/Checkout/AddressManagement/AddAddressModel/StateDatabases/USStates.js

```diff
@@ -0,0 +1,59 @@
+export const USStates = () => {
```

Comment:
I have a suggestion, as this is getting repeated, better to create an array of object containing state and state code
`usState = [{stateCode:"AL", state: "Alabama"}]`
Now you can map this array

One more thing you can do if you have multiple countries:
```
countryAndStates = { 
    usState: [{stateCode:"AL", state: "Alabama"}],
    indianState:  [{stateCode:"BH", state: "Bihar"}] 
}
```
Now you can keep this data in database file and this way mapping would be easier, whenever want to add or remove anything it would be easier to do

Reply:
+1